rfcs/2020-07-21-2744-vicscript.md

```diff
@@ -93,42 +93,28 @@ multiplied = number * 7
 Perform assignments conditionally with an `if` statement:
 
 ```coffee
-sorted_foo = if foo.type() == "array" { foo.sort() }
+sorted_foo = if foo.type() == "array" { foo.sort() } else { foo }
 ```
 
-And use a `match` statement for pattern matching:
+```
+In:  {"foo":"not an array"}
+Out: {"sorted_foo":"not an array"}'
 
-```coffee
-new_doc = match doc {
-  type == "article" => article
-  type == "comment" => comment
-  _ => this
-}
+In:  {"foo":["c","a","d","b"]}
+Out: {"sorted_foo":["a","b","c","d"]}
 ```
 
+
 Use a wealth of methods on values in order to perform common mutations on them:
 
 ```coffee
 sorted = foo.sort()
 uppercase = bar.uppercase()
```

Comment:
I think it would be worth clarifying if these operations mutate the underlying field. It appears they don't but it's not clear. Actually, that might be worth stating as an overall guiding principle.